script/script.go

```diff
@@ -677,6 +683,30 @@ func setRequestHeader(f filters.FilterContext) func(*lua.LState) int {
 	}
 }
 
+func addRequestHeader(f filters.FilterContext) func(*lua.LState) int {
```

Comment:
It should be possible to generalize this and reuse for request and response headers but for simplicity I decided to introduce separate functions to follow the style of existing single value setters/getters and also because they return closures.